### ISSUE TITLE
Signature field sanitization

### ISSUE THREAD

**Issue by CyrusV5** [epochtalk/epochtalk#522]
The signature field is vulnerable to persistent XSS attacks.
POC code injected: `{{constructor.constructor('alert("Hi! :)")')()}}`

Trying to edit the signature gives an error: `There was an error updating user <user>`

**Comment by akinsey** [comment]
Fixed here: https://github.com/epochtalk/epochtalk/commit/2a7e3835741c31de80bf549df3f28fe1cce83cbd

You will have to change branches to `fixes-2019`

1.  git checkout fixes-2019`
2. `git pull`
3. Edit package.json in root of `epochtalk` project. Change the following line:
```
"epochtalk-core-pg": "^1.0.9",
```
To this:
```
"epochtalk-core-pg": "https://github.com/epochtalk/core-pg.git",
```
4. Run `yarn install` in the root of `epochtalk

5. Checkout the latest of the `epoch` db migrations project

6. Within the root of the `epoch` project, run `mix ecto.migrate`

This branch has a bunch of other fixes as well, just let me know if you have any issues!

Thanks Cyrus!

@CyrusV5

**Comment by unenglishable** [comment]
resolved by #534
